### ISSUE TITLE
Publish crank docker image to an image registry?

### ISSUE THREAD

**Issue by dazinator** [dotnet/crank#377]
Would you consider publishing the crank docker image to some registry for easier consumption?

**Comment by sebastienros** [comment]
Yes, I had started the process on the mcr registry. Not on a short term plan though due to other priorities.
